intg/src/main/java/org/apache/atlas/model/lineage/AtlasLineageOnDemandInfo.java

```diff
@@ -0,0 +1,407 @@
+    }
+
+    public Set<LineageRelation> getRelations() {
+        return relations;
+    }
+
+    public void setRelations(Set<LineageRelation> relations) {
+        this.relations = relations;
+    }
+
+    public LineageDirection getLineageDirection() {
+        return lineageDirection;
+    }
+
+    public void setLineageDirection(LineageDirection lineageDirection) {
+        this.lineageDirection = lineageDirection;
+    }
+
+    public int getLineageDepth() {
+        return lineageDepth;
+    }
+
+    public void setLineageDepth(int lineageDepth) {
+        this.lineageDepth = lineageDepth;
+    }
+
+    public Set<String> getVisitedEdges() {
+        return visitedEdges;
+    }
+
+    public void setVisitedEdges(Set<String> visitedEdges) {
+        this.visitedEdges = visitedEdges;
+    }
+
+    public Set<String> getSkippedEdges() {
+        return skippedEdges;
+    }
+
+    public void setSkippedEdges(Set<String> skippedEdges) {
+        this.skippedEdges = skippedEdges;
+    }
+
+    public Map<String, LineageInfoOnDemand> getRelationsOnDemand() {
+        return relationsOnDemand;
+    }
+
+    public void setRelationsOnDemand(Map<String, LineageInfoOnDemand> relationsOnDemand) {
+        this.relationsOnDemand = relationsOnDemand;
+    }
+    public Map<String, LineageOnDemandConstraints> getLineageOnDemandPayload() {
+        return lineageOnDemandPayload;
+    }
+
+    public void setLineageOnDemandPayload(Map<String, LineageOnDemandConstraints> lineageOnDemandPayload) {
+        this.lineageOnDemandPayload = lineageOnDemandPayload;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        AtlasLineageOnDemandInfo that = (AtlasLineageOnDemandInfo) o;
+        return lineageDepth == that.lineageDepth &&
+                Objects.equals(baseEntityGuid, that.baseEntityGuid) &&
+                lineageDirection == that.lineageDirection &&
+                Objects.equals(guidEntityMap, that.guidEntityMap) &&
+                Objects.equals(relations, that.relations);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(baseEntityGuid, lineageDirection, lineageDepth, guidEntityMap, relations);
+    }
+
+    @Override
+    public String toString() {
+        return "AtlasLineageInfo{" +
+                "baseEntityGuid=" + baseEntityGuid +
+                ", guidEntityMap=" + guidEntityMap +
+                ", relations=" + relations +
+                ", lineageDirection=" + lineageDirection +
+                ", lineageDepth=" + lineageDepth +
+                '}';
+    }
+
+    @JsonAutoDetect(getterVisibility = PUBLIC_ONLY, setterVisibility = PUBLIC_ONLY, fieldVisibility = NONE)
+    @JsonSerialize(include = JsonSerialize.Inclusion.NON_NULL)
+    @JsonIgnoreProperties(ignoreUnknown = true, value = {"inputRelationsReachedLimit", "outputRelationsReachedLimit"})
```

Comment:
inputRelationsReachedLimit, outputRelationsReachedLimit - no variables with this name

Reply:
addressed